mobile-widgets/qml/StartPage.qml

```diff
@@ -32,12 +26,32 @@ Kirigami.ScrollablePage {
 			text: manager.startPageText
 			wrapMode: Text.WordWrap
 		}
-		CloudCredentials {
-			id: cloudCredentials
+		Kirigami.Label {
+			id: explanationTextBasic
+			visible: !showPin
 			Layout.fillWidth: true
 			Layout.margins: Kirigami.Units.gridUnit
-			Layout.topMargin: 0
-			property int headingLevel: 3
+			Layout.topMargin: Kirigami.Units.gridUnit * 3
+			text: qsTr("To use Subsurface-mobile with Subsurface cloud storage, please enter your cloud credentials.\n\n") +
+				qsTr("If this is the first time you use Subsurface cloud storage, enter a valid email (all lower case) " +
+				"and a password of your choice (letters and numbers).\n\n") +
+				qsTr("To use Subsurface-mobile only with local data on this device, select " +
+				"the no cloud buttton above.")
+			wrapMode: Text.WordWrap
+		}
+		Kirigami.Label {
+			id: explanationTextPin
+			visible: showPin
+			Layout.fillWidth: true
+			Layout.margins: Kirigami.Units.gridUnit
+			Layout.topMargin: Kirigami.Units.gridUnit * 3
+			text: qsTr("Thank you for registering with Subsurface. We sent ") + manager.cloudUserName +
+				qsTr(" a PIN code to complete the registration. ") +
+				qsTr("If you do not receive an email from us within 15 minutes, please check " +
+					 "the correct spelling of your email address and your spam box first.\n\n" +
+					 "In case of any problems regarding cloud account setup, please contact us " +
+					 "at our user forum \(https://subsurface-divelog.org/user-forum/\).\n\n")
+			wrapMode: Text.WordWrap
```

Comment:
same as above comment.

you can also use this for arguments to `qsTr()`:
````
qsTr("File %1 of %2").arg(counter).arg(total)
````
srouce: http://doc.qt.io/qt-5/qtquick-internationalization.html#4-use-op-op-x-to-insert-parameters-into-a-string